### PR TITLE
Add action code, prepare v1 release

### PR DESCRIPTION
**Pull request by awendt**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+outputs:
+  current-projects:
+    description: "Projects a pull request has cards on"
+    value: ${{ steps.get-current-projects.outputs.current-projects }}
+runs:
+  using: composite
+
+
+  steps:
+    - id: get-current-projects
```
Comment (jtsaito) [inline action.yml]: The number of newlines looks non-uniform. Maybe:

```suggestion
outputs:
  current-projects:
    description: "Projects a pull request has cards on"
    value: ${{ steps.get-current-projects.outputs.current-projects }}

runs:
  using: composite

  steps:
    - id: get-current-projects
```

**Comment by jtsaito** [comment]
@awendt Please add the link to the associated story.

**Review by jtsaito — APPROVED** [review]

**Comment by awendt** [comment]
[Stop comment spam via whatchanged bot](https://trello.com/c/S7Zasgim/3250-stop-comment-spam-via-whatchanged-bot)

**Review by fmasuhr — APPROVED** [review]

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                  }
+                }
+              }
+            }' | jq -c '
+                .data.repository.pullRequest.projectCards.nodes
+              | map({
+                databaseId,
+                name: .project.name
+              })'
+        )
```
Comment (jansiwy) [inline action.yml]: let's unify the pipes (e.g. always prefix, not infix) and the brackets (incl. multi-line quotes, e.g. closing bracket/quote always in the same column where the command with the opening bracket/quote starts):

```suggestion
        CURRENT_PROJECTS=$(
          hub api --paginate graphql -f query='
            query {
              repository(owner: "${{ github.event.repository.owner.login }}", name: "${{ github.event.repository.name }}") {
                pullRequest(number: ${{ github.event.pull_request.number }}) {
                  projectCards {
                    nodes {
                      project {
                        name
                      }
                      databaseId
                    }
                  }
                }
              }
            }
          ' \
          | jq -c '
              .data.repository.pullRequest.projectCards.nodes
              | map({
                  databaseId,
                  name: .project.name
                })
            '
        )
```

**Review by jansiwy — APPROVED** [review]
apart from that: +B

**Pull request merged**